### PR TITLE
Refactor/general cleanup

### LINKED ISSUE #54: redirect error page to /error or something

### LINKED ISSUE #61: cant scroll on feelings log?

### PR DESCRIPTION
**Pull request by espressoGoddess**
### Files Changed:
- `App.js`, `FeelingsLog.css`, `FeelingsLog.js`

### What Has Changed: 
- This styles the feelingsLog table so that it is scrollable.
- This truncates the activities in the feelingsLog (adds ellipse) so its stays one width.
- This redirects the error page to path `/404` for funsies.

### Closes: 
Closes #61, closes #54

### CODE REVIEW

File: client/src/Components/FeelingsLog/FeelingsLog.css [hunk 1]
```diff
@@ -24,6 +25,49 @@ th {
...
+  
+  tbody::-webkit-scrollbar-track {
+    background-color: var(--element-bkgd-color); 
+  }
+  
+  tbody::-webkit-scrollbar-thumb {
+    background-color: var(--element-bkgd-color);
+    border-radius: 20px;
+    border: solid rgba(255,255,255, .5);
+  }
```
Comment (KatherineBlaine) [inline client/src/Components/FeelingsLog/FeelingsLog.css:73]: Excited to see how this styling looks! Love that you added some scrollbar styling.

File: client/src/Components/FeelingsLog/FeelingsLog.js [hunk 2]
```diff
@@ -20,7 +20,7 @@ const FeelingsLog = ({logs}) => {
     return (<tr key={log.entry_id}>
         <td>{log.date.toLocaleString(DateTime.DATE_MED)}</td>
         <td>{toTitleCase(log.feeling)}</td>
-        <td>{log.activity}</td>
+        <td className='truncate'>{log.activity}</td>
```
Comment (KatherineBlaine) [inline client/src/Components/FeelingsLog/FeelingsLog.js:23]: Love! Cant wait to see.

**Review by KatherineBlaine — APPROVED** [review]
This looks great! Thanks for doing this styling cleanup.

File: client/src/Components/App/App.js [hunk 3]
```diff
@@ -72,7 +72,8 @@ const App = () => {
           return <JournalEntry journal={journal}/>
         }}/>
         <Route exact path='/how-you-felt'><FeelingsLog logs={userLogs}/></Route>
-        <Route path='*'><NotFound /></Route>
+        <Route exact path='/404'><NotFound /></Route>
+        <Route path='*'><Redirect to='/404'/></Route>
```
Comment (chrissycooper) [inline client/src/Components/App/App.js:76]: love this

File: client/src/Components/FeelingsLog/FeelingsLog.css (hunk 1 above)
Comment (chrissycooper) [reply client/src/Components/FeelingsLog/FeelingsLog.css:73]: Thank you for working on this Amber!

**Review by chrissycooper — APPROVED** [review]

File: client/src/Components/App/App.js (hunk 3 above)
Comment (jfogiato) [inline client/src/Components/App/App.js:76]: +1 to CRC! Love this! Would love to hear more about your understanding of how `<Redirect>` works at the next standup!

File: client/src/Components/App/App.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import React, {useState, useEffect} from 'react';
-import { Route, Switch } from 'react-router-dom';
+import { Route, Switch, Redirect } from 'react-router-dom';
```
Comment (jfogiato) [inline client/src/Components/App/App.js:2]: SKRRRRRT

File: client/src/Components/FeelingsLog/FeelingsLog.css (hunk 1 above)
Comment (jfogiato) [reply client/src/Components/FeelingsLog/FeelingsLog.css:73]: +1 - loved the sneak peek we got!

File: client/src/Components/FeelingsLog/FeelingsLog.css [hunk 5]
```diff
@@ -24,6 +25,49 @@ th {
...
 }
 
-.feelingsLogs tbody tr:nth-of-type(odd) {
-    background-color: var(--background-color);
-}
\ No newline at end of file
+.truncate {
+    max-width: 450px;
+    overflow: hidden;
+    text-overflow: ellipsis;
```
Comment (jfogiato) [inline client/src/Components/FeelingsLog/FeelingsLog.css:31]: ...

**Review by jfogiato — APPROVED** [review]
Love all of these additions. The redirect is primo, the added styling for the scroll bar is *Chefs kiss*. Thanks for your work on this, Amber!

**Pull request merged**